mobile/components/Card.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { StyleProp, ViewStyle, View} from "react-native";
+import { Card as TamaguiCard, CardProps as TamaguiCardProps} from 'tamagui'
+import { tokens } from "../theme/tokens";
+
+export interface CardProps extends TamaguiCardProps {
+    customStyle?: StyleProp<ViewStyle>
```

Comment:
name it style, like the default, instead of customStyle, it's more intuitive that way

Reply:
done